docs/koa-vs-express.md

```diff
@@ -39,7 +39,7 @@
 ## Does Koa replace Connect?
 
   No, just a different take on similar functionality
-  now that generators allow us to write code with less
```

Comment:
It would be better to keep `less` over `fewer`!

Reply:
fewer refers to countable, which the number of callbacks would be.

Reply:
this change is fine by me